src/set_theory/ordinal/basic.lean

```diff
@@ -957,6 +957,13 @@ theorem ord_card_le (o : ordinal) : o.card.ord ≤ o := gc_ord_card.l_u_le _
 
 lemma lt_ord_succ_card (o : ordinal) : o < (succ o.card).ord := lt_ord.2 $ lt_succ _
 
+/--
+The converse of the following lemma is false (take, for instance `i = ω+1` and `κ = ℵ₀`).
```

Comment:
The docstring is something which shows up as a tooltip when using the lemma. So it should explain the content of the lemma. Additional comments such that the fact that the converse is false are definitely useful, but the first goal is to explain the content of the lemma.

Reply:
This file has already been reached by *the tide*. What would be the best course of action now? Either,
 - move the change to the point of first use (perhaps with a TODO sign to later fix it), therefore leaving this file unmodified; or
 - keep the edit as it stands, and when (if) the changes are approved, create a parallel PR to mathlib4?

Reply:
> This file has already been reached by _the tide_. What would be the best course of action now? Either,
> 
> * move the change to the point of first use (perhaps with a TODO sign to later fix it), therefore leaving this file unmodified

I decided to go this way. I didn't put the TODO sign after all, but I keep a personal record for tidying things up later.